The job **`x86_64-gnu-llvm-13`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/3570012111/jobs/6000567942) [(plain)](https://github.com/rust-lang/rust/commit/02e2ae5853abf4a9626d2fb2e3c6070beec6773d/checks/9759283006/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

---- [ui] src/test/ui/suggestions/suggest_print_over_printf.rs stdout ----
diff of stderr:

1 error[E0425]: cannot find function `printf` in this scope
-   --> $DIR/seggest_print_over_printf.rs:5:5
3    |
4 LL |     printf("%d", x);
5    |     ^^^^^^ not found in this scope

---
To only update this specific test, also pass `--test-args suggestions/suggest_print_over_printf.rs`

error: 1 errors occurred comparing output.
status: exit status: 1
command: "/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" "/checkout/src/test/ui/suggestions/suggest_print_over_printf.rs" "-Zthreads=1" "--target=x86_64-unknown-linux-gnu" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/suggestions/suggest_print_over_printf" "-A" "unused" "-Crpath" "-Cdebuginfo=0" "-Lnative=/checkout/obj/build/x86_64-unknown-linux-gnu/native/rust-test-helpers" "-L" "/checkout/obj/build/x86_64-unknown-linux-gnu/test/ui/suggestions/suggest_print_over_printf/auxiliary"
Some tests failed in compiletest suite=ui mode=ui host=x86_64-unknown-linux-gnu target=x86_64-unknown-linux-gnu
stdout: none
--- stderr -------------------------------
error[E0425]: cannot find function `printf` in this scope
   |
LL |     printf("%d", x);
   |     ^^^^^^ not found in this scope
   |
   |
help: you may have meant to use the `print` macro
   |
LL |     print!("%d", x);

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
```

</details>

--- a/compiler/rustc_resolve/src/late/diagnostics.rs
+++ b/compiler/rustc_resolve/src/late/diagnostics.rs
@@ -282,7 +282,7 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                             "you may want to use a bool value instead",
                             format!("{}", item_typo),
                         ))
-                    // FIXME(vicnenzopalazzo): make the check smarter,
+                    // FIXME(vincenzopalazzo): make the check smarter,
                     // and maybe expand with levenshtein distance checks
                     } else if item_str.as_str() == "printf" {
                         Some((

--- a/src/test/ui/suggestions/suggest_print_over_printf.rs
+++ b/src/test/ui/suggestions/suggest_print_over_printf.rs
@@ -1,5 +1,4 @@
-// Suggest to a user to use the print macros
-// instead to use the printf.
+// Suggest print macro when user erroneously uses printf
 
 fn main() {
     let x = 4;

--- a/src/test/ui/suggestions/suggest_print_over_printf.stderr
+++ b/src/test/ui/suggestions/suggest_print_over_printf.stderr
@@ -1,5 +1,5 @@
 error[E0425]: cannot find function `printf` in this scope
-  --> $DIR/seggest_print_over_printf.rs:6:5
+  --> $DIR/suggest_print_over_printf.rs:5:5
    |
 LL |     printf("%d", x);
    |     ^^^^^^ not found in this scope

